transformer_engine/pytorch/attention.py

```diff
@@ -1062,37 +1106,78 @@ def _rotate_half(x: torch.Tensor) -> torch.Tensor:
 def apply_rotary_pos_emb(
     t: torch.Tensor,
     freqs: torch.Tensor,
+    tensor_format: str = "sbhd",
     fused: bool = False,
+    transpose_output_memory: bool = False,
```

Comment:
Is `transpose_output_memory` kwarg needed?

Reply:
Removed.